components/Context/StorageProvider.tsx

```
@@ -129,72 +124,43 @@
             setWalletTransactionUpdateStatus(WalletTransactionsStatus.NONE);
-          }
-          if (noErr) await saveToDisk();
-          resolve();
-        });
+            resolve();
+          });
+        } catch (error) {
+          console.error(error);
+          reject(error);
+        }
       });
```

Biome

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)